### PR TITLE
Features/algorithms

### PR DESCRIPTION
**Pull request by kellyburg**
Created a file for game play component and set up initial state for first and second selection.

### CODE REVIEW

**Comment by serpient** [comment]
I'm not sure what's happening , but formatting in the file looks strange .
for example, the lines are not aligned

File: src/VocabGamePlay.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+
+class VocabGamePlay extends Component{
+    constructor(){
+		super();
```
Comment (serpient) [inline src/VocabGamePlay.jsx]: constructor and super should have props passed in
```
constructor(props) {
   super(props)
}
```

**Review by serpient — CHANGES_REQUESTED** [review]

**Review by kellyburg — COMMENTED** [review]
Is the formatting better? I did cmd+k+f and it realigned the brackets.

File: src/VocabGamePlay.jsx (hunk 1 above)
Comment (kellyburg) [reply src/VocabGamePlay.jsx]: done!

**Comment by serpient** [comment]
much better!!! one more thing, please remove the extra lines between content 💃 
i know its just a styling thing, but it keeps it compact as it grows

**Review by serpient — APPROVED** [review]

**Pull request merged**